pypeit/spectrographs/magellan_fire.py

```diff
@@ -176,7 +176,8 @@ def default_pypeit_par(cls):
         par['calibrations']['slitedges']['pca_order'] = 3
 
         # Model entire slit
-        par['reduce']['extraction']['model_full_slit'] = True # local sky subtraction operates on entire slit
+        par['reduce']['extraction']['model_full_slit'] = True  # local sky subtraction operates on entire slit
+        par['reduce']['findobj']['maxnumber'] = 1  # Slit is narrow so allow one object per order
```

Comment:
Same as with GNIRS.  If we go with this, we should add to the instrument docs that this is set and tell people how to increase in the PypeIt file.